### PR TITLE
fix(slides): support CSV multi-value for --slide-id in screenshot

### PR DESCRIPTION
**Pull request by BD-ZERO**
--slide-id used the cobra StringArray flag type, which only accepts repeated flags and does not split comma-separated values, unlike --slide-number (int_array -> cobra IntSlice) which already supported CSV input. This made the two selector flags inconsistent.

Switch --slide-id to the string_slice flag type (cobra StringSlice), which natively supports both comma-separated and repeated values, and update the flag readers from StrArray to StrSlice. normalizeSlideIDs already trims/dedupes/filters blanks, and
validateSlidesScreenshotSelectorLimit already caps the combined selector count, so both continue to apply unchanged to CSV input.

Add tests covering --slide-id CSV parsing, whitespace/duplicate normalization, and the >10 selector limit via CSV, mirroring the existing --slide-number coverage.

## Changes

- Change 1
- Change 2

## Test Plan

- [ ] Unit tests pass
- [ ] Manual local verification confirms the `lark-cli <domain> <command>` flow works as expected

## Related Issues

- None

## Summary

* **Bug Fixes**
  * Improved `--slide-id` handling for the Slides Screenshot shortcut when provided as a comma-separated list.
  * Slide ID lists are now normalized by trimming whitespace, removing empty entries, and deduplicating while preserving order.
  * When more than 10 slide IDs are provided, screenshot requests now fail with a clear “request at most 10 pages at a time” validation message.
  * Validation now prevents combining `--content` with `--slide-number` (conflict message unchanged).

### CODE REVIEW

File: shortcuts/slides/slides_screenshot_test.go [hunk 1]
```diff
@@ -188,6 +188,136 @@ func TestSlidesScreenshotListBySlideNumber(t *testing.T) {
...
+
+	path1 := filepath.Join(dir, defaultSlidesScreenshotDir, "pres_abc_slide_1.png")
+	if _, err := os.ReadFile(path1); err != nil {
+		t.Fatalf("read first CSV slide screenshot: %v", err)
+	}
+	path2 := filepath.Join(dir, defaultSlidesScreenshotDir, "pres_abc_slide_2.png")
+	if _, err := os.ReadFile(path2); err != nil {
+		t.Fatalf("read second CSV slide screenshot: %v", err)
+	}
+}
```
[the remark this answers is not in the document]
Comment (fangshuyu-768) [reply shortcuts/slides/slides_screenshot_test.go:245]: Reopening because this remains unaddressed at `6f46877`. None of the three new screenshot tests sets `LARKSUITE_CLI_CONFIG_DIR` before creating the test factory, despite the repository testing requirement in `AGENTS.md`. Please add `t.Setenv("LARKSUITE_CLI_CONFIG_DIR", t.TempDir())` to each affected test so local configuration state cannot leak into or be modified by the suite.

File: shortcuts/slides/slides_screenshot_test.go [hunk 2]
```diff
@@ -185,6 +185,136 @@ func TestSlidesScreenshotListBySlideNumber(t *testing.T) {
...
+						"format":   1,
+						"data":     base64.StdEncoding.EncodeToString([]byte("png-bytes-2")),
+					},
+				},
+			},
+		},
+	}
+	reg.Register(stub)
+
+	err := runSlidesShortcut(t, f, stdout, SlidesScreenshot, []string{
```
Comment (fangshuyu-768) [inline shortcuts/slides/slides_screenshot_test.go:217]: **[P3] Add the required dry-run E2E coverage for this flag-type change.** `AGENTS.md` requires a dry-run E2E for every shortcut change. These package tests exercise the mounted shortcut, but there is no `tests/cli_e2e` case that pins CSV parsing through the built CLI and verifies the emitted `slide_ids` request body. Please add a dry-run case using `--slide-id slide_1,slide_2`.

File: shortcuts/slides/slides_screenshot.go [hunk 3]
```diff
@@ -43,7 +43,7 @@ var SlidesScreenshot = common.Shortcut{
 	AuthTypes:         []string{"user", "bot"},
 	Flags: []common.Flag{
 		{Name: "presentation", Desc: "xml_presentation_id, slides URL, or wiki URL that resolves to slides; list mode only"},
-		{Name: "slide-id", Type: "string_array", Desc: "slide page identifier (repeat for multiple slides; max 10 pages per request)"},
+		{Name: "slide-id", Type: "string_slice", Desc: "slide page identifier (repeat or comma-separate for multiple slides; max 10 pages per request)"},
```
Comment (fangshuyu-768) [inline shortcuts/slides/slides_screenshot.go]: **[P3] Document the new CSV syntax in the Slides skill reference.** The CLI help now advertises comma-separated selectors, but `skills/lark-slides/references/lark-slides-screenshot.md` still says that multi-page requests require repeating `--slide-id`. Since agents use that reference as the command contract, the new behavior remains undiscoverable there. Please update the flag table and add a CSV example.

**Review by fangshuyu-768 — APPROVED** [review]
